test/Fitup.js

```diff
@@ -15,31 +15,31 @@ contract("Fitup", accounts => {
     before( async () => {
         fitup = await Fitup.deployed();
     });
-    it('creates a bet', async () => {
+    beforeEach( async () => {
         await fitup.createBet(
             organisation, { 
             from: account_one,
             value: one_eth
         });
+    });
+    it('creates a bet', async () => {
         bet = await fitup.getBet(account_one);
         expect(bet.organisation).to.equal(organisation);
         expect(bet.amount).to.equal(one_eth);
     });
     it('lost a bet', async() => {
-        const before = Number(ethers.utils.formatEther(await web3.eth.getBalance(organisation)));
-        await fitup.payoutBet(false, account_one);
-        const after = Number(ethers.utils.formatEther(await web3.eth.getBalance(organisation)));
+        let {before, after} = await beforeAfterPayout(organisation, false) // bet lost
         expect(before).to.be.lessThan(after);
     });
     it('won a bet', async() => {
-        await fitup.createBet(
-            organisation, { 
-            from: account_one,
-            value: one_eth
-        });
-        const before = Number(ethers.utils.formatEther(await web3.eth.getBalance(account_one)));
-        await fitup.payoutBet(true, account_one);
-        const after = Number(ethers.utils.formatEther(await web3.eth.getBalance(account_one)));
+        let {before, after} = await beforeAfterPayout(account_one, true) // bet won
         expect(before).to.be.lessThan(after);
     });
-})
\ No newline at end of file
+    async function beforeAfterPayout(from, result) {
```

Comment:
strange function "beforeAfter..." 😀

Reply:
Haha true maybe beforeAndAfterPayout is a better naming?